source/platforms/databricks.md

```diff
@@ -63,9 +63,70 @@ gdf
+
+Running `pip install dask-databricks` should launch a dask scheduler in the driver node and workers on remaining nodes in just a few minutes.
+
+### Create init-script
+
+To get started, you must first configure an [initialization script](https://docs.databricks.com/en/init-scripts/index.html) to install `dask`, RAPIDS libraries and all other dependencies for your project.
+
+Databricks recommends using [cluster-scoped](https://docs.databricks.com/en/init-scripts/cluster-scoped.html) init scripts stored in the workspace files.
+
+Navigate to the top-left **Workspace** tab and click on your **Home** directory then select **Add** > **File** from the menu. Create an `init.sh` script with contents:
+
+```bash
+#!/bin/bash
+set -e
+
+# The Databricks Python directory isn't on the path in
+# databricksruntime/gpu-tensorflow:cuda11.8 for some reason
+export PATH="/databricks/python/bin:$PATH"
+
+# Install RAPIDS (cudf & dask-cudf) and dask-databricks
+/databricks/python/bin/pip install --extra-index-url=https://pypi.nvidia.com \
+      cudf-cu11 \  # installs cudf
+      dask[complete] \
+      dask-cudf-cu11  \ #installs dask-cudf
+      dask-cuda=={rapids_version} \
+      dask-databricks
+
+# Start the Dask cluster with CUDA workers
+dask databricks run --cuda
 
 ```
 
-## Databricks Spark
+**Note**: To launch a dask cluster with GPU workers, you must parse in `--cuda` flag option when running the command, otherwise the script will launch standard dask workers by default.
 
-You can also use the RAPIDS Accelerator for Apache Spark 3.x on Databricks. See the [Spark RAPIDS documentation](https://nvidia.github.io/spark-rapids/docs/get-started/getting-started-databricks.html) for more information.
+### Launch Dask cluster
+
+Once your script is ready, follow the same [instructions](#launch-cluster) to launch a **Multi-node** Databricks cluster.
+
+After docker setup in **Advanced Options**, switch to the **Init Scripts** tab and add the file path to the init-script in your Workspace directory starting with `/Users/<user-name>/<script-name>.sh`.
+
+You can also configure cluster log delivery in the **Logging** tab, which will write the init script logs to DBFS in a subdirectory called `dbfs:/cluster-logs/<cluster-id>/init_scripts/`. Refer to [docs](https://docs.databricks.com/en/init-scripts/logs.html) for more information.
+
+![Screenshot of init script](../images/databricks-dask-init-script.png)
+
+Now you should be able to select a "GPU-Accelerated" instance for both **Worker** and **Driver** nodes.
+
+![Screenshot of driver worker node](../images/databricks-worker-driver-node.png)
+
+### Connect to Client
+
+To test RAPIDS, Connect to the dask client and submit tasks.
+
+![Screenshot of dask-client](../images/databricks-mnmg-dask-client.png)
+
+![Screenshot of dask-client](../images/databricks-mnmg-dask-example.png)
```

Comment:
I would probably just replace this screenshot with a code cell. Copy/paste makes users lives way easier.